dao.md

```diff
@@ -0,0 +1,40 @@
+# 🦞 Senior Blockchain Developer - OceanDAO
+[Ocean Protocol](https://www.oceanprotocol.com/) has a vibrant, expanding ecosystem of engineers, data scientists, neuroscientists, ambassadors, podcasters, that are build traction across 5 pillars of growth, receiving grants and funding to deliver results. To serve them better, we’re hiring!
+
+[OceanDAO](https://www.oceanprotocol.com/dao) is a key component that brings decentralization and scale to participants that are driving traction into the Ocean ecosystem. We’re growing the OceanDAO team. If you love to work with crypto, to build and to engage deeply with an enthusiastic community, we want to hear from you.
+
+
+## Responsibilities
+* You will design & build for OceanDAO core software, from the level of blockchain to the level of dapps. Integrating governance, treasury management, viral mechanisms, and whatever else is needed for OceanDAO to be concretized.
+* The scope of the software is well defined but may evolve over time as DAOs are nascent, e.g. integrating decentralized courts, and decentralizing funding mechanisms. You will co-plan and co-design the product with the OceanDAO team in an agile, open-source development process.
+
+
+## Minimum Qualifications
+* 6+ months of Solidity experience
+* 3+ years backend experience, including NodeJS
```

Comment:
I would order the experience needed based on years, in reverse order. To 4+ years of software experience to be the first one, then 3+ years of backend the second one and the 6+ months, the last one